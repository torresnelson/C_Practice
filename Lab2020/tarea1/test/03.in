esClave 15
asociar 15 5.2
esClave 14
esClave 15
valor 15
desasociar 15
esClave 15
asociar 95 24.0
valor 95
asociar 85 2.0
valor 85
valor 95
desasociar 85
esClave 85
valor 95
asociar 65 3.0
asociar 25 67.0
asociar 35 -12.0
asociar 75 2.5
asociar 45 2.1
asociar 55 -24.0
valor 55
valor 65
valor 95
desasociar 55
desasociar 95
desasociar 65
desasociar 25
desasociar 35
esClave 95
esClave 15
valor 75
Fin
