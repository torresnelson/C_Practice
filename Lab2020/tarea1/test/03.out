1>15 no es clave. 
2>Asociar.
3>14 no es clave. 
4>15 es clave. 
5>15 está asociado a 5.20. 
6>Desasociar. 
7>15 no es clave. 
8>Asociar.
9>95 está asociado a 24.00. 
10>Asociar.
11>85 está asociado a 2.00. 
12>95 está asociado a 24.00. 
13>Desasociar. 
14>85 no es clave. 
15>95 está asociado a 24.00. 
16>Asociar.
17>Asociar.
18>Asociar.
19>Asociar.
20>Asociar.
21>Asociar.
22>55 está asociado a -24.00. 
23>65 está asociado a 3.00. 
24>95 está asociado a 24.00. 
25>Desasociar. 
26>Desasociar. 
27>Desasociar. 
28>Desasociar. 
29>Desasociar. 
30>95 no es clave. 
31>15 no es clave. 
32>75 está asociado a 2.50. 
33>Fin.
